source/quick-start-legacy.txt

```diff
@@ -0,0 +1,127 @@
+===========================
```

Comment:
For a quick start, do we even need to talk about the legacy API at all?  If someone is using legacy, it's only because they already have an existing application that uses it, and therefore doesn't need a quick start guide in the first place.

Reply:
I considered that, and was thinking of the situation in which a developer is introduced to an existing application/service that uses the legacy API and wants to learn from the ground up. I can remove it if you think it's not necessary.

I am curious what your opinion is on how much information about the legacy API that you wanted to keep.
 
For example, in the Usage Examples section, should we offer examples using both the current and legacy API, or add a note that mentions the difference in the MongoClient classes (along with any other differences)?

Reply:
> was thinking of the situation in which a developer is introduced to an existing application/service that uses the legacy API and wants to learn from the ground up

I think it's fine in this case to learn the new API.  If the application is existing already, generally they are not going to be having to construct a MongoClient themselves anyway.  It will already be created somewhere and stashed in a global variable.   So I think you can remove.

> I am curious what your opinion is on how much information about the legacy API that you wanted to keep.

For the reference documentation, I think we should continue to give examples using both the new and old MongoClient, since the old is still widely used.